openfl-workspace/workspace/plan/defaults/aggregator.yaml

```diff
@@ -1,3 +1,4 @@
 template : openfl.component.Aggregator
 settings :
     db_store_rounds : 2
+    persist_checkpoint: True
```

Comment:
Could you also test E2E with `persist_checkpoint: False`, just to make sure the original flow still works?

Reply:
I did ofc :)